bundles/thunk_bundle.js

```diff
@@ -6945,6 +6945,31 @@ module.exports = (() => {
               return parentFormula.execute(params, executionContext);
             }
             break;
+          case "CellAutocomplete" /* CellAutocomplete */:
```

Comment:
This is unusual. There's hackery here both to set up the execution context with additional properties and to observe which properties are read and stuff them secretly into the formula result before returning